### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by VickyP13**

### CODE REVIEW

**Review by anonymoustuna — COMMENTED** [review]
LGTM

**Review by anonymoustuna — APPROVED** [review]

**Review by Smirnova-Katya — APPROVED** [review]

File: src/test/java/testQaSoCalVictoria.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class testQaSoCalVictoria {
```
Comment (Smirnova-Katya) [inline src/test/java/testQaSoCalVictoria.java:8]: Class testQaSoCalVictoria should be extended from BaseTest

File: src/test/java/testQaSoCalVictoria.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class testQaSoCalVictoria {
+    @Test
+    public void itemInCard() throws InterruptedException {
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\vilun\\Downloads\\chromedriver_win32\\chromedriver.exe");
+
+        WebDriver driver = new ChromeDriver();
```
Comment (Smirnova-Katya) [inline src/test/java/testQaSoCalVictoria.java:13]: It's not necessary cause we have a driver manager. You can use getDriver() from BaseTest

File: src/test/java/testQaSoCalVictoria.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        WebElement userName = driver.findElement(By.id("user-name"));
+        WebElement password = driver.findElement(By.id("password"));
+        WebElement loginBtn = driver.findElement(By.id("login-button"));
+
+        userName.sendKeys("standard_user");
+        password.sendKeys("secret_sauce");
+        loginBtn.click();
+
+        WebElement addToCartBtn = driver.findElement(By.id("add-to-cart-sauce-labs-backpack"));
+        addToCartBtn.click();
```
Comment (Smirnova-Katya) [inline src/test/java/testQaSoCalVictoria.java:25]: If you use a variable just once, there is no point to create a variable
```suggestion
        driver.findElement(By.id("add-to-cart-sauce-labs-backpack")).click();

```

File: src/test/java/testQaSoCalVictoria.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        WebElement addedToCardBtn = driver.findElement(By.xpath("//span[@class='shopping_cart_badge']"));
+        addedToCardBtn.click();
```
Comment (Smirnova-Katya) [inline src/test/java/testQaSoCalVictoria.java:27]: The same as an above

**Review by Smirnova-Katya — CHANGES_REQUESTED** [review]

File: src/test/java/testQaSoCalVictoria.java (hunk 4 above)
Comment (VickyP13) [reply src/test/java/testQaSoCalVictoria.java:27]: changed

File: src/test/java/testQaSoCalVictoria.java (hunk 3 above)
Comment (VickyP13) [reply src/test/java/testQaSoCalVictoria.java:25]: changed

File: src/test/java/testQaSoCalVictoria.java (hunk 1 above)
Comment (VickyP13) [reply src/test/java/testQaSoCalVictoria.java:8]: ok

**Pull request closed without merging**